Charts: describe 'shift' method (#4812)
DevExpress/devextreme-documentation: api-reference/10 UI Components/BaseChart/7 Chart Elements/Label/3 Methods/shift(x_y).md

---
---
##### shortDescription
Moves label to the specified location.

##### return: Object
The label's instance.

##### param(x): Number
The `x` coordinate.

##### param(y): Number
The `y` coordinate.

---

> The origin is at the upper left corner of the Chart. The coordinates represent the distance between a label and the origin and should be positive numbers.

Reviewer: All comments are pretty minor by themselves, but together they warrant an edit: 

- We try to substitute the word "represent" with a synonym where possible. "Represent" is a bit vague. 
- "And should be" - creates an additional clause that is in the very end, but refers to the very first noun of the sentence. So I re-arrange it all a little bit. 
- If I measure distance, I would mention "origin" first. Looks like it's natural to start from zeroes. So I'm changing "between label and origin" to "between origin and label". 

```suggestion
The origin is at the upper-left corner of the Chart. The coordinates are positive numbers that specify the distance between the origin and the label.
```

> The following example moves label of a point of a bar Chart to the position above the bar:

Reviewer: I want to change the part that repeats "of a" in close succession: "label of a point of a bar Chart". 

```suggestion
The following example customizes a Bar Chart. The code moves a label to the top of the corresponding bar. 
```

---
---
##### shortDescription
Moves label to the specified location.

##### return: Object
The label's instance.

##### param(x): Number
The `x` coordinate.

##### param(y): Number
The `y` coordinate.

---
The origin is at the upper-left corner of the Chart. The coordinates are positive numbers that specify the distance between the origin and the label.

> The following example customizes a Bar Chart. The code moves a label to the top of the corresponding bar. 

Reviewer 2: ```suggestion
The following example customizes a Bar Chart. The code moves the label to the top of the corresponding bar. 
```
